api/openapi/specs/api.yaml

```diff
@@ -172,6 +172,42 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /meshes/{mesh}/{serviceType}/{serviceName}/_hostnames:
+    get:
+      operationId: inspect-hostnames
+      summary: Returns hostnames for service
+      description: Returns hostnames for a service
+      tags: ["Inspect"]
+      parameters:
+        - in: path
+          name: mesh
+          example: default
+          schema:
+            type: string
+          required: true
+          description: The mesh the service is part of
+        - in: path
+          name: serviceType
+          example: meshservices
+          schema:
+            type: string
+            enum: [meshservices, meshmultizoneservices, meshexternalservices]
+          required: true
+          description: The type of the service
+        - in: path
+          name: serviceName
```

Comment:
We agree that this doesn't support URIs yet?

Reply:
URIs?

Reply:
unique resource identifier (`<mesh>_<name>_<namespace>_<zone>_<shortType>_<optionalSection>`)

Reply:
then no. At least not for now